### PR TITLE
Fix storybook tests

### PR DESCRIPTION
**Pull request by thomtrp**
Before 
Capture d’écran 2024-10-31 à 16 58 34

After
Capture d’écran 2024-10-31 à 16 58 14

Before
Capture d’écran 2024-10-31 à 16 57 59

After
Capture d’écran 2024-10-31 à 16 57 29

### CODE REVIEW

File: packages/twenty-front/src/modules/settings/data-model/fields/forms/relation/components/SettingsDataModelFieldRelationForm.tsx [hunk 1]
```diff
@@ -101,7 +101,10 @@ export const SettingsDataModelFieldRelationForm = ({
   });
 
   const selectedObjectMetadataItem = findObjectMetadataItemById(
-    watchFormValue('relation.objectMetadataId'),
+    watchFormValue(
+      'relation.objectMetadataId',
```
Comment (Weiko) [inline packages/twenty-front/src/modules/settings/data-model/fields/forms/relation/components/SettingsDataModelFieldRelationForm.tsx:105]: Is this change intended in this pr @thomtrp ?

**Review by Weiko — APPROVED** [review]
LGTM thank you @thomtrp !

File: packages/twenty-front/src/modules/settings/data-model/fields/forms/relation/components/SettingsDataModelFieldRelationForm.tsx (hunk 1 above)
Comment (thomtrp) [reply packages/twenty-front/src/modules/settings/data-model/fields/forms/relation/components/SettingsDataModelFieldRelationForm.tsx:105]: Yes it fixes the second test
